### DISCUSSION TITLE
Is it possible to customize month names in RangePicker so they become full length.

### DISCUSSION THREAD

**Discussion by aliendrippin** [ant-design/ant-design#41098] [Q&A] [answered]
For example Dec, Jan, Feb etc. are expected to be December, January, February.

**Comment by Wxh16144** [comment, marked as answer]
~~Direct replacement is not yet supported. If you are using 5.x and using dayjs, I have given you a modified demo https://codesandbox.io/s/xiu-gai-ri-qi-yue-fen-xian-shi-5nburg?file=/demo.tsx, which may be a bit rough and hopefully will solve your problem for now.~~

~~But I don't think this solution is quite right and we need to discuss it more.~~

---

Sorry, there was a problem with my original answer, you can achieve it by: [online demo](https://codesandbox.io/s/xiu-gai-yue-fen-de-ge-shi-2-707hdb?file=/demo.tsx)

```tsx
import React from "react";
import { DatePicker, Space } from "antd";
import en_US from "antd/locale/en_US";

const { RangePicker } = DatePicker;

const locale = {
  ...en_US.DatePicker,
  lang: {
    ...en_US.DatePicker.lang,
    monthFormat: "MMMM"
  }
};

const App: React.FC = () => (
  <Space direction="vertical">
    <RangePicker locale={locale} />
    <DatePicker locale={locale} />
  </Space>
);

export default App;
```

**Reply by aliendrippin** [reply]
Thank you, that suits perfectly to me.
